charts/nri-prometheus/README.md

```diff
@@ -1,6 +1,6 @@
 # nri-prometheus
 
-![Version: 2.1.7](https://img.shields.io/badge/Version-2.1.7-informational?style=flat-square) ![AppVersion: 2.16.2](https://img.shields.io/badge/AppVersion-2.16.2-informational?style=flat-square)
+![Version: 2.1.8](https://img.shields.io/badge/Version-2.1.8-informational?style=flat-square) ![AppVersion: 2.16.2](https://img.shields.io/badge/AppVersion-2.16.2-informational?style=flat-square)
```

Comment:
I think we are removing this from the template .

Reply:
It's still there, I'm merging this and we can remove it in a different PR. Thanks!